openbharatocr/ocr/pan.py

```diff
@@ -63,16 +70,26 @@ def extract_dob(input):
 def extract_pan_details(image_path):
 
     image = Image.open(image_path)
-
     extracted_text = pytesseract.image_to_string(image)
-
-    full_name, fathers_name = extract_names(extracted_text)
+    
+    format = imghdr.what(image_path)
+    if format != "jpeg":
+        image.save("image.jpg", "JPEG")
+        
+        converted_image = Image.open("image.jpg")
+        converted_image_text = pytesseract.image_to_string(converted_image)
+        
+        extracted_text += converted_image_text
+
+    names = extract_all_names(extracted_text)
+    full_name = names[0] if len(names) > 0 else ""
+    fathers_name =  names[1] if len(names) > 1 else ""
     dob = extract_dob(extracted_text)
     pan_number = extract_pan(extracted_text)
 
     return {
         "Full Name": full_name,
-        "Father's Name": fathers_name,
+        "Father's/Mother's Name": fathers_name,
```

Comment:
fathers_name is making confusion 
you can use `parents name`

Reply:
Changed the variable name.